### ISSUE TITLE
Model Implementation

### ISSUE THREAD

**Issue by MrRincon** [MrRincon/ReelFeel_NLP_RNN#30]
- [x] Importing NLP Preprocessing Libraries
- [x] Part of Speech  (POS), Cleaning Data, and Tokenization
- [x] Importing Word2Vec Embedding Model
- [x] Importing TensorFlow Libraries for the RNN
- [x] Data Split, Recurrent Neural Network (RNN), Dropout, Rectified Linear Unit
- [x] Executing RNN, Early Stopping, Model Training, and History Visualization
- [x] Evaluation and Insights
